src/Helpers/SolrUpdate.php

```diff
@@ -35,6 +36,9 @@ class SolrUpdate
      */
     public function updateItems($items, $type, $index = null)
     {
+        if (!$index) {
```

Comment:
Why disallow the null index? It's on purpose to loop all indexes

Reply:
I haven't rebased the code, my branch is behind master  :)